README.md

```
@@ -215,6 +216,10 @@ structured-coding-with-ai  (canonical AI methodology)
 
 ---
 
+## Code Quality
+
+<img src="scorecard.png" width="100%">
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)